core/domain/classifier_services.py

```diff
@@ -305,9 +287,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, interaction_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status, job_id=None):
```

Comment:
Yes, I agree with having separate functions for 'create' and 'update'. Would that solve the problem?